service/app/controllers/bulkapimanagement/BaseBulkUploadController.java

```diff
@@ -91,7 +91,11 @@
     if (validateFileZize) {
       checkFileSize(byteArray, objectType);
     }
-    reqObj.setOperation(operation);
+    if (map.get("operation") != null) {
+      reqObj.setOperation("userBulkSelfDeclared");
```

Comment:
I find this string used at least in 2-3 places. Can you add to keys please?
Also, pls rename this to "userDeclarations" or likewise.